src/lib/drizzle.ts

```diff
@@ -1,10 +1,44 @@
 import { drizzle } from "drizzle-orm/postgres-js";
+import { sql } from "drizzle-orm";
 import postgres from "postgres";
 import * as schema from "./schema";
 
 if (!process.env.DATABASE_URL) {
   throw new Error("database url is undefined");
 }
 
-export const sql = postgres(process.env.DATABASE_URL);
-export const db = drizzle(sql, { schema });
+export const connection = postgres(process.env.DATABASE_URL);
+export const db = drizzle(connection, { schema });
+
+// player queries
+export type NewPlayer = typeof schema.player.$inferInsert;
+export const insertPlayer = async (newPlayer: NewPlayer) => {
+  return db.insert(schema.player).values(newPlayer);
+};
```

Comment:
Maybe we create a `data/` folder inside of `app/` that contains files like `game.ts` or `player.ts` where we can write the queries from the db. Then this file can simply be used for configuration and the types